frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { match as RouteMatch } from 'react-router';
+import { useTranslation } from 'react-i18next';
+import { Wizard, WizardStep } from '@patternfly/react-core';
+import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
+import { ListKind } from '@console/internal/module/k8s';
+import { CreateStorageSystemFooter } from './footer';
+import { CreateStorageSystemHeader } from './header';
+import { BackingStorage, createSteps } from './create-storage-system-steps';
+import { initialState, reducer, WizardReducer } from './reducer';
+import { StepsId, StorageClusterIdentifier } from '../../constants/create-storage-system';
+import { StorageSystemKind } from '../../types';
+import { StorageSystemModel } from '../../models';
+
+const CreateStorageSystem: React.FC<CreateStorageSystemProps> = ({ match }) => {
+  const { t } = useTranslation();
+  const [state, dispatch] = React.useReducer<WizardReducer>(reducer, initialState);
+  const [ssList, ssLoaded, ssLoadError] = useK8sGet<ListKind<StorageSystemKind>>(
+    StorageSystemModel,
+  );
+
+  const { url } = match;
+
+  let wizardSteps: WizardStep[] = [];
+
+  if (ssLoaded && !ssLoadError) {
+    const hasOCS = ssList?.items?.some((ss) => ss.spec.kind === StorageClusterIdentifier);
+    wizardSteps = ssLoaded && !ssLoadError ? createSteps(t, state, hasOCS) : [];
```

Comment:
```suggestion
    wizardSteps = createSteps(t, state, hasOCS);
```
Isn't the outer `if` statement guarding for this?

Reply:
yeah missed.